policies/project-director-election-process.md

```diff
@@ -0,0 +1,60 @@
+# Project Director Election Process
+
+## Aims
+
+The purpose of this policy is to outline a clear and efficient process for electing new Project Directors to represent the Rust project on the board of the Rust Foundation. The process aims to achieve the following goals:
+
+* Select candidates who fulfill the outlined candidate criteria, focusing on shared power and new leadership rather than solely prioritizing the best candidate.
+    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference.
```

Comment:
This reads a bit jargony. If one has not heard the phrase "overlap of tolerance" or "overlap of preference" this sentence might be very hard to understand. Can we phrase this in a way that is clearer even if it requires being more verbose?

Reply:
I'll second this. I almost made a similar comment on my review but couldn't think of terminology I liked better. Tolerance in particular struck me as being used in a technical sense that doesn't really match how I usually think of it in normal conversation. I think the thing that tolerance is meant to convey here is that we want to stress finding a solution everyone can live (i.e. tolerate) with over getting anyone's first choice (preference).

Maybe just a clarifying sentence explaining what we mean would help, like this?

```suggestion
    * Strive for tolerance and inclusivity, aiming for the overlap of tolerance rather than the overlap of preference. In other words, we want to prioritize a selection that everyone can accept over selecting anyone's first choice.
```